x-pack/plugins/apm/public/components/shared/Stacktrace/index.tsx

```diff
@@ -40,14 +41,21 @@ export function Stacktrace({ stackframes = [], codeLanguage }: Props) {
       {groups.map((group, i) => {
         // library frame
         if (group.isLibraryFrame) {
-          const initialVisiblity = groups.length === 1; // if there is only a single group it should be visible initially
+          const hasMultipleStackframes = group.stackframes.length > 1;
+          const hasLeadingSpacer = hasMultipleStackframes && i !== 0;
+          const hasTrailingSpacer =
+            hasMultipleStackframes && i !== groups.length - 1;
           return (
-            <LibraryStackFrames
-              key={i}
-              initialVisiblity={initialVisiblity}
-              stackframes={group.stackframes}
-              codeLanguage={codeLanguage}
-            />
+            <Fragment>
+              {hasLeadingSpacer && <EuiSpacer size="m" />}
+              <LibraryStackFrames
```

Comment:
@ogupte `key` should be moved to fragment to avoid the error seen in https://github.com/elastic/kibana/issues/29037